packages/nameguard-react/src/components/ChatModal/ChatModal.tsx

```diff
@@ -40,9 +40,7 @@ export const ChatModal = forwardRef(
             >
               <IconButton
                 iconPosition="right"
-                icon={
-                  <ArrowTopRightOnSquareIcon className="w-5 h-5 fill-current" />
-                }
+                icon={<ArrowUpRightIcon className="w-5 h-5 fill-current" />}
```

Comment:
We shouldn't use anything from HeroIcons on this. It should be exactly the same icon that we're using for external links in other `IconButton`.